### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Reinmar** [dlibby-/highlight#11]
I wonder if and how the semantics of created highlights should be exposed to assistive technologies. 

For instance – when highlights would be used to render spellchecking squiggles we should also have a way to tell the that this piece of text is misspelled. I don't know how the selection itself is rendered in the Accessibility Object Model, but I think that highlights should be able to use the same mechanism.

I'd imagine that the easiest solution may be to allow defining a highlight's title. For a misspelled fragment of text it could be set to "misspelled" (that's at least what VoiceOver reads).

**Comment by Reinmar** [comment]
cc @Comandeer

**Comment by Comandeer** [comment]
In the simplest case something like `HighlightRange#label` should be enough. However, if you already referenced AOM, there is proposal for [virtual accessibility nodes](https://github.com/WICG/aom/blob/gh-pages/explainer.md#virtual-accessibility-nodes), that are dedicated just for this case: exposing non-element nodes in accessibility tree. In such case `HighlightRange` interface should be updated in following manner:

```webidl
partial interface HighlightRange {
	AccessibleNode attachAccessibleRoot();
}
```

The only part that it's worrying me a little is the fact that VANs are behaving a little like pseudoelements (they could have dimensions and position).

**Comment by gked** [comment]
moved this issue to https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/73
